frontend/config/environment.js

```diff
@@ -63,10 +64,10 @@ module.exports = function(environment) {
   };
 
   if (environment === 'development') {
-    ENV['ember-simple-auth-token'].refreshAccessTokens = false;
```

Comment:
Any reason this was removed? I don't think the auth API is currently configured to support automatic access token refreshing. (https://github.com/jpadilla/ember-simple-auth-token#json-web-token-authenticator-1)

Reply:
because it is set to false in the default configuration (see line 14 in this file), so this is just unnecessary (and thus a lil confusing IMO)

Reply:
Oh, word 👍

Reply:
Unless it should be "true" in production configuration?

Reply:
No, it shouldn't, because automatic token refreshes haven't been implemented in the API. (As far as I understand it, it's a way of keeping a user signed in indefinitely.)